tests/unit/TagFieldTest.php

```diff
@@ -0,0 +1,395 @@
+<?php
+
+class TagFieldTest extends SapphireTest {
+	/**
+	 * @var string
+	 */
+	public static $fixture_file = 'tagfield/tests/unit/TagFieldTest.yml';
+
+	/**
+	 * @var array
+	 */
+	protected $extraDataObjects = array(
+		'TagFieldTest_BlogTag',
+		'TagFieldTest_BlogPost',
+	);
+
+	/**
+	 * @test
+	 */
+	public function itSavesLinksToExistingTagsOnExistingRecords() {
```

Comment:
Wouldn't it be easier to just stick to the PHPUnit convention and prefix the methods with `test`? Then you don't need annotations for each of them. The sentence structure can stay the same.